### DISCUSSION TITLE
Fatal error: Uncaught GuzzleHttp\Exception\ClientException: Client error: `GET https://daily-flyer-srl-69f86.firebaseio.com/Admin` resulted in a `404 Not Found`

### DISCUSSION THREAD

**Discussion by ghost** [kreait/firebase-php#545] [Q&A] [answered]
hello,
 I am trying to use firebase authentication to login user and access data from firebase. I'm having an issue.

Fatal error: Uncaught GuzzleHttp\Exception\ClientException: Client error: `GET https://daily-flyer-srl-69f86.firebaseio.com/Admin` resulted in a `404 Not Found` response: in C:\xampp\htdocs\Daily Flyer SRL\vendor\guzzlehttp\guzzle\src\Exception\RequestException.php:113 Stack trace: #0 C:\xampp\htdocs\Daily Flyer SRL\vendor\guzzlehttp\guzzle\src\Middleware.php(69): GuzzleHttp\Exception\RequestException::create(Object(GuzzleHttp\Psr7\Request), Object(GuzzleHttp\Psr7\Response), NULL, Array, NULL) #1 C:\xampp\htdocs\Daily Flyer SRL\vendor\guzzlehttp\promises\src\Promise.php(204): GuzzleHttp\Middleware::GuzzleHttp\{closure}(Object(GuzzleHttp\Psr7\Response)) #2 C:\xampp\htdocs\Daily Flyer SRL\vendor\guzzlehttp\promises\src\Promise.php(153): GuzzleHttp\Promise\Promise::callHandler(1, Object(GuzzleHttp\Psr7\Response), NULL) #3 C:\xampp\htdocs\Daily Flyer SRL\vendor\guzzlehttp\promises\src\TaskQueue.php(48): GuzzleHttp\Promise\Promise::GuzzleHttp\Promise\{closure}() #4 C:\xampp\htdocs\Daily Flyer SRL\vendor\guzzlehttp\promises\s in C:\xampp\htdocs\Daily Flyer SRL\vendor\kreait\firebase-php\src\Firebase\Exception\DatabaseApiExceptionConverter.php on line 62

```php
if (isset($_POST['Login'])) 
{
  $email    = $_POST['admin_email'];
  $loginstatus=0;
  $pass    = $_POST['admin_password'];
  if($email != '' && $pass != '')
  {
  	$user =$auth->signInWithEmailAndPassword($email, $pass);
  	if($user)
  	{
      $ref="Admin/";
        $udata=$database->getReference($ref)->getValue();
        if ($udata)
              {
              #products loop
                foreach ($udata as $key => $userdata)
                    {
                      if ($userdata)
                      {
                        #now get values
                        $uemail=$userdata['Email'];
                        if($email==$uemail)
                        {
                          #set session variables and break out of loop
                          $loginstatus=1;
                          $_SESSION['user_id']= $key;
                          $_SESSION['username']=$userdata['Name'];
                          $_SESSION['user_email']=$userdata['Email'];
                          header("Location:dashboard.php");
                        }
                      }
                  }
                  if ($loginstatus==0)
                  {
                    #set error message user is not admin
                    header("Location:index.php");
                  }
              }
              else
              {
                #show error messsage user is not admin
                header("Location:index.php");
              }
    }
  	else
  	{
  		#show error messsage user is not registered or user details incorrect
  		header("Location:index.php");
  	}
  }
  else
  {
  	#show error user details are empty
    header("Location: index.php");
  }
}
```

**Comment by jeromegamez** [comment, marked as answer]
Hi 👋! I edited your post for readability (code formatting).

Please check at https://console.firebase.google.com/project/_/database/ for the name of your database (it's the URL that looks like `https://<some string>.firebaseio.com`.

If it's not `https://daily-flyer-srl-69f86.firebaseio.com` (this is the default URL generated from the SDK based on the project ID of your project), please use the `withDatabaseUri()` method on the factory:

```php
$factory = (new Factory)
    ->withServiceAccount('path/to/your/service_account.json')
    ->withDatabaseUri('<URL from your Firebase console>');

$database = $factory->createDatabase();
```

**Reply by ghost** [reply]
Thank you very much, sir. It worked for me.

I was struggling to fix this error but you saved my life. Thank you

**Reply by jeromegamez** [reply]
You're welcome!

**Reply by jewel-rana** [reply]
Thanks a lot
